Sources/Cells/TabbyCell.swift

```diff
@@ -0,0 +1,94 @@
+import UIKit
+
+class TabbyCell: UICollectionViewCell {
+
+  static let reusableIdentifier = "TabbyCellReusableIdentifier"
+
+  lazy var imageView: UIImageView = {
+    let imageView = UIImageView()
+    imageView.contentMode = .ScaleAspectFit
+
+    return imageView
+  }()
+
+  lazy var label: UILabel = {
+    let label = UILabel()
+    label.font = Constant.Font.title
+    label.textColor = Constant.Color.disabled
+
+    return label
+  }()
+
+  // MARK: - Configuration
+
+  func configureCell(item: TabbyBarItem, selected: Bool = false) {
+    let color = selected ? Constant.Color.selected : Constant.Color.disabled
+
+    imageView.image = item.image?.imageWithRenderingMode(.AlwaysTemplate)
+    imageView.tintColor = color
+
+    label.text = item.controller.title
+    label.textColor = color
+
+    if selected {
+      TabbyAnimation.animate(imageView, kind: item.animation)
+    }
+
+    handleBehaviors(selected)
+    setupConstraints()
+  }
+
+  // MARK: - Helper methods
+
+  func handleBehaviors(selected: Bool) {
+    switch Constant.Behavior.labelVisibility {
+    case .Invisible:
+      label.alpha = 0
+    case .ActiveVisible:
+      label.alpha = selected ? 1 : 0
+    default:
+      label.alpha = 1
+    }
+  }
+
+  // MARK: - Constraints
+
+  func setupConstraints() {
+    let offset: CGFloat = label.alpha == 1 ? 8 : 0
```

Comment:
A beautiful magic number... Maybe it should be a configurable constant. Thought about it.